compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4825,6 +4843,11 @@ private STNode createMissingSimpleObjectField(STNode metadata, List<STNode> qual
                 simpleNameRef, identifier, emptyNode, emptyNode, semicolon);
     }
 
+    private STNode createMissingSimpleObjectField() {
+        List<STNode> qualifiers = new ArrayList<>();
+        return createMissingSimpleObjectField(null,  qualifiers, false);
```

Comment:
Instead of directly using null, let's use `STNodeFactory.createEmptyNode()` to represent empty metadata.